client/profile-wizard/index.js

```diff
@@ -215,6 +221,16 @@ class ProfileWizard extends Component {
 		}
 	}
 
+	async skipProfiler() {
+		const { updateProfileItems } = this.props;
+		await updateProfileItems( {
+			skipped: true,
+			step: this.getCurrentStep().key,
+		} );
+		recordEvent( 'storeprofiler_store_details_skip' );
+		window.location = getAdminLink( 'admin.php?page=wc-admin' );
```

Comment:
Same here.

Also, does it make sense to `await` the return of the profile items request here? Can we just navigate to the homescreen right away?

Reply:
And I changed it here too 😀

Even though it looks like the `await` is useless, it is necessary to be sure the `updateProfileItems` has finished before the redirection. Otherwise, when the home screen is loading, the check could fail and it would redirect the user back to the OBW.